src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -35,12 +39,32 @@ function ProviderSelectionField({
   const loadingProviders =
     requestStatus === FETCH_STATUS.loading ||
     requestStatus === FETCH_STATUS.notStarted;
+
+  const loadingLocations = requestLocationStatus === FETCH_STATUS.loading;
+
   const providerSelected = 'id' in formData;
+  const key = sortMethod || FACILITY_SORT_METHODS.distanceFromResidential;
```

Comment:
I think we set the sort method by default in the initial redux state, so I don't think you need to do this.

Reply:
Done.